slidingwindow.h

```diff
@@ -1201,14 +1201,22 @@ void ConvolutionInputGenerator_kernel1(
 		hls::stream<ap_uint<SIMD*Input_precision> > & in,
 		hls::stream<ap_uint<SIMD*Input_precision> > & out,
 		const unsigned int numReps) {
-static_assert(IFMChannels % SIMD == 0, "");
-	for (unsigned int im=0; im<numReps; im++) {
-		for (unsigned int y = 0; y < IFMDim; y++) {
-			for (unsigned int x = 0; x < IFMDim; x++) {
-				for (unsigned int count_simd =0; count_simd < IFMChannels/SIMD; count_simd++) {
+static_assert(IFMChannels % SIMD == 0);
```

Comment:
Please, add back in an assertion message to avoid triggering compiler warnings in C++14.